react-app/src/components/projects/ProjectNotificationsTable.tsx

```diff
@@ -77,12 +81,62 @@ const ProjectNotificationsTable = (props: ProjectNotificationsTableProps) => {
 
   if (!props.rows) return <></>;
 
+  // Prepare values for Enhanced Referral Notification fields
+  const initalERN = props.rows.find(
+    (row) => row.subject === 'ACTION REQUIRED - Notification of Surplus Real Property',
+  );
+  const thirtyDayERN = props.rows.find(
+    (row) =>
+      row.subject ===
+      'Notification of Surplus Real Property - 30 Day Reminder Notification of Surplus Real Property',
+  );
+  const sixtyDayERN = props.rows.find(
+    (row) =>
+      row.subject ===
+      'Notification of Surplus Real Property - 60 Day Reminder Notification of Surplus Real Property',
+  );
+  const nintyDayERN = props.rows.find(
+    (row) =>
+      row.subject ===
+      'Notification of Surplus Real Property - Completion of 90 Day Enhanced Referral Period for Notification of Surplus Real Property',
+  );
+
   return !props.rows ? (
     <Box display={'flex'} justifyContent={'center'}>
       <Typography>No notifications were sent.</Typography>
     </Box>
   ) : (
     <>
+      <LocalizationProvider dateAdapter={AdapterDayjs}>
+        <Typography variant="h6">Enhanced Referral Notification Dates</Typography>
+        <Box gap={1} display={'inline-flex'} mb={3} mt={2}>
```

Comment:
I think these looking so greyed out next to the other elements looks a bit off. We could make some of the elements black? Just a suggestion.

```
<LocalizationProvider dateAdapter={AdapterDayjs}>
        <Typography variant="h6">Enhanced Referral Notification Dates</Typography>
        <Box
          gap={1}
          display={'inline-flex'}
          mb={3}
          mt={2}
          sx={{
            '& .MuiInputBase-root.Mui-disabled': {
              '& > fieldset': {
                borderColor: 'rgba(0,0,0)',
              },
            },
            '& .MuiFormLabel-root.MuiInputLabel-root': {
              color: 'rgba(0, 0, 0)',
            }
          }}
        >
        ....

Reply:
Good point. Updated.